test/fluentd-forward.sh

```diff
@@ -10,74 +10,175 @@ FLUENTD_WAIT_TIME=${FLUENTD_WAIT_TIME:-$(( 2 * minute ))}
 
 os::test::junit::declare_suite_start "test/fluentd-forward"
 
+extra_artifacts=$ARTIFACT_DIR/fluentd-forward-artifacts.txt
```

Comment:
use `artifact_out` and `artifact_log`

Reply:
This is a backport (source: https://github.com/openshift/origin-aggregated-logging/pull/926)
Should I fix the newer versions, too?

Reply:
arg - sorry, I missed the review of the earlier PRs - then no, don't fix it here

instead, submit another PR (later) to fix the test in master and cherry pick from there

Reply:
Yep.  I've just submitted this PR: https://github.com/openshift/origin-aggregated-logging/pull/946